packages/std/extra/live_update/scripts/live_update_from_npm_packages.nu

```diff
@@ -1,4 +1,12 @@
-let version = http get $'https://registry.npmjs.org/($env.packageName)/latest'
+# Get project metadata
+mut project = $env.project
+  | from json
+
+# Retrieve the latest release information from npm registry
```

Comment:
The naming `npm` respects the one of this [PR](https://github.com/brioche-dev/brioche-packages/pull/783)